src/utils/childrenValueInputValidation.js

```diff
@@ -1,10 +1,8 @@
 import React from 'react';
 import singlePropFrom from 'react-prop-types/lib/singlePropFrom';
 
-const propList = ['children', 'value'];
-
 export default function valueValidation(props, propName, componentName) {
-  let error = singlePropFrom(propList)(props, propName, componentName);
+  let error = singlePropFrom('children', 'value')(props, propName, componentName);
```

Comment:
this is perfectly fine, but you could have also done: `singlePropFrom(...propList)` :)